freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -149,3 +158,39 @@ def _filter_bundles_by_pinned_related_images(self, bundle_image_nvrs):
                     continue
                 ret_bundle_images_nvrs.add(nvr)
         return ret_bundle_images_nvrs
+
+    def get_published_original_nvr(self, session, rebuilt_nvr):
+        """
+        Search for an original build, that has been built and published to a
+            repository, and get original_nvr from it
+
+        :param session: database session
```

Comment:
Why are you passing in session as an argument but using `db.session` directly? I would just remove this argument.